src/locales/en.json5

```diff
@@ -1,10 +1,42 @@
 // action: This is the text for an action, for example, the text of a button.
 // field: This is the text of a form field.
 // header: This is the text of a table column header.
-
 // tab: This is the text of a navigation tab.
 
 {
+  "title": {
```

Comment:
This sounds good! Do you have suggestions on how to approach this?

1. Should everything just go into `src/locales/en.json5`?
2. What about the previously-translated page titles for these components? How can I preserve that translation work?

Reply:
I do think that each title (or title part) will need a corresponding message in `en.json5`, so if a title is defined in a component, it should be moved to `en.json5`. Unfortunately, that will result in some work for translators, because the new messages in `en.json5` will appear as new strings to translate. For that reason, it's generally a good idea to avoid moving a message unnecessarily. That said, Transifex makes the process much easier, because Transifex will remember the previous translation and will offer it as a suggestion. So if there's a reason to move a message, it's also OK to do so.

One thing to look out for is that some of the messages used in components actually already live in `en.json5`. For example, the text "App Users" for the App Users tab lives in `en.json5` at `resource.appUsers`. I think the easiest thing to do would be to keep those messages where they are, then use them in `routes.js`. How does that sound? You could also considering adding some messages to the `resource` object rather than the `title` object. Like maybe "Projects" could go under `resource.projects`?